webpack.config.babel.js

```diff
@@ -138,9 +153,10 @@ const basePlugins = [
     // Increment BASE_32_VERSION with each release following Base32 notation, i.e AA -> AB
     // Do it only when we introduce a breaking change between SDK and cross device client
     // ref: https://en.wikipedia.org/wiki/Base32
-    'BASE_32_VERSION' : 'AC',
+    'BASE_32_VERSION' : 'AD',
```

Comment:
I don't think we introduced a breaking change between SDK and cross device client, did we?

Reply:
Most probably we didn't, but when I give it a second thought, it seems to be safer to bump with every release. The cost of bumping is much less than cost of forgetting to bump in case we miss something (especially when another person is doing the release). What do you think?

Reply:
Yes, that feels safer but I don't see the point of having a separate versioning for something that we bump all the times when the package version changes. Probably we should consider getting rid of it.